src/ethereum/arrow_glacier/vm/gas.py

```diff
@@ -81,18 +81,18 @@ class ExtendMemory:
 @dataclass
 class MessageCallGas:
     """
-    Define the gas cost and stipend for executing the call opcodes.
+    Define the gas cost and gas given to the sub-call for
+    executing the call opcodes.
 
     `cost`: `ethereum.base_types.Uint`
-        The non-refundable portion of gas reserved for executing the
-        call opcode.
-    `stipend`: `ethereum.base_types.Uint`
+        The gas required to execute the call opcode.
+    `sub_call`: `ethereum.base_types.Uint`
         The portion of gas available to sub-calls that is refundable
-        if not consumed
+        if not consumed.
     """
 
     cost: Uint
```

Comment:
Would it make sense to also rename `cost` to `total` or something similar? Looks like it excludes the memory expansion cost though, and that might be a tad misleading.

Reply:
- Added in a comment that it excludes memory expansion to be explicit.
- Keeping `cost` in the variable name is good imo since it is added to some other `cost` afterwards, maybe `total_cost` would work? 

https://github.com/ethereum/execution-specs/blob/6c337f7c579e38170813d10cf4685afe0f04c869/src/ethereum/arrow_glacier/vm/instructions/system.py#L353